### PR TITLE
VDS-2185, VDS-2239, VDS-2289: Members list with full-text search and filter by status

### PR DESCRIPTION
**Pull request by Pav-el**
Description: 

--

QA-test: VDS-2187, VDS-2241, VDS-2291

Demo-test: VDS-2188, VDS-2242, VDS-2292

Download artifact URL: ghcr.io/virtocommerce/demo-xapi-app:1.0.0-pr-72-vds-2239-72a8d62e

### CODE REVIEW

File: src/app/modules/members/components/members-list/members-list-filter-bar/input-values.model.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export interface InputValues {
```
Comment (ilyawzrd) [inline src/app/modules/members/components/members-list/members-list-filter-bar/input-values.model.ts]: It's not used anywhere, you can remove it

**Review by ilyawzrd — CHANGES_REQUESTED** [review]

**Review by ilyawzrd — APPROVED** [review]

File: src/app/modules/members/components/members-list/members-list-filter-bar/members-list-filter-bar.component.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      selectInputValue: this.selectInputValue,
+    });
+  }
+
+  onSearchButtonClick(): void {
+    this.filterChange.emit({
+      searchFilterValue: this.searchFilterValue,
+      selectInputValue: this.selectInputValue,
+    });
+  }
```
Comment (asvishnyakov) [inline src/app/modules/members/components/members-list/members-list-filter-bar/members-list-filter-bar.component.ts]: Duplicate emits can be moved to single reused method

File: src/app/modules/members/components/members-list/members-list-table/member-status/member-status.component.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  @extend .base;
+  color: $success;
+  border-color: $success;
+}
+
+.rejected {
+  @extend .base;
+  color: $danger;
+  border-color: $danger;
+}
```
Comment (asvishnyakov) [inline src/app/modules/members/components/members-list/members-list-table/member-status/member-status.component.scss]: May be replaced with `border-2 border-XXX color-XXX` & padding classes _possibly_

Comment (ilyawzrd) [reply src/app/modules/members/components/members-list/members-list-table/member-status/member-status.component.scss]: @Pav-el `class="border-2" [ngClass]="{'border-success text-success': status === 'new' ......}"` try something like that

**Review by ilyawzrd — APPROVED** [review]

**Review by asvishnyakov — APPROVED** [review]

**Pull request merged**